tests/platform_tests/conftest.py

```diff
@@ -416,10 +416,11 @@ def verify_required_events(duthost, event_counters, timing_data, verification_er
             observed_end_count = timing_data.get(
                 key, {}).get(pattern, {}).get("End count", 0)
             expected_count = event_counters.get(pattern)
-            # If we're checking PORT_READY, and there are 0 port state change messages captured instead of however many
-            # was expected, treat it as a success. Some platforms (Mellanox, Dell S6100) have 0, some platforms (Arista
-            #  050cx3) have however many ports are up.
-            if observed_start_count != expected_count and (pattern != 'PORT_READY' or observed_start_count != 0):
+            # If we're checking PORT_READY, allow any number of PORT_READY messages between 0 and the number of ports.
```

Comment:
Can this be improved to check for duplicates per port? Duplicates likely indicate a flap and that is unexpected.

Reply:
Change to be done as part of an enhancement by @saiarcot895